tests/unit/models/transactions/test_xchain_create_bridge.py

```diff
@@ -0,0 +1,166 @@
+from unittest import TestCase
+
+from xrpl.models import (
+    XRP,
+    IssuedCurrency,
+    XChainBridge,
+    XChainCreateBridge,
+    XRPLModelException,
+)
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_ACCOUNT2 = "rpZc4mVfWUif9CRoHRKKcmhu1nx2xktxBo"
+_FEE = "0.00001"
+_SEQUENCE = 19048
+
+_ISSUER = "rGWrZyQqhTp9Xu7G5Pkayo7bXjH4k4QYpf"
+
+_GENESIS = "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh"
+
+
+class TestXChainCreateBridge(TestCase):
+    def test_successful_xrp_xrp_bridge(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=XRP(),
+            issuing_chain_door=_GENESIS,
+            issuing_chain_issue=XRP(),
+        )
+        XChainCreateBridge(
+            account=_ACCOUNT,
+            fee=_FEE,
+            sequence=_SEQUENCE,
+            xchain_bridge=bridge,
+            signature_reward="200",
+            min_account_create_amount="1000000",
+        )
+
+    def test_successful_iou_iou_bridge(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=IssuedCurrency(currency="USD", issuer=_ISSUER),
+            issuing_chain_door=_ACCOUNT2,
+            issuing_chain_issue=IssuedCurrency(currency="USD", issuer=_ACCOUNT2),
+        )
+        XChainCreateBridge(
+            account=_ACCOUNT,
+            fee=_FEE,
+            sequence=_SEQUENCE,
+            xchain_bridge=bridge,
+            signature_reward="200",
+        )
+
+    def test_same_door_accounts(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=IssuedCurrency(currency="USD", issuer=_ISSUER),
+            issuing_chain_door=_ACCOUNT,
+            issuing_chain_issue=IssuedCurrency(currency="USD", issuer=_ACCOUNT),
+        )
+        with self.assertRaises(XRPLModelException):
+            XChainCreateBridge(
+                account=_ACCOUNT,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                xchain_bridge=bridge,
+                signature_reward="200",
+            )
+
+    def test_xrp_iou_bridge(self):
+        bridge = XChainBridge(
+            locking_chain_door=_ACCOUNT,
+            locking_chain_issue=XRP(),
+            issuing_chain_door=_ACCOUNT,
+            issuing_chain_issue=IssuedCurrency(currency="USD", issuer=_ACCOUNT),
```

Comment:
Is this supposed to be the same account twice?

Reply:
(Happens several times in this file)

Reply:
Yes, that's how it's designed. The issuing chain issue has to be the same currency as the locking chain issue, and issued from the issuing chain door (because it needs to be able to mint/burn the wrapped token).